client/my-sites/plugins/plugin-action/style.scss

```diff
@@ -33,26 +39,37 @@
 }
 
 .plugin-action__label {
-	color: $gray;
-	font-size: 11px;
+	font-size: 12px;
 	line-height: 16px;
 	margin-right: 8px;
 	vertical-align: top;
-	text-transform: uppercase;
 	cursor: pointer;
 
+	@include breakpoint( '>1040px' ) {
```

Comment:
@iamtakashi Changing the styles of the `plugin-action` and `plugin-activate-toggle` this way breaks layout of other places where these toggles are used.

First of them is the `PluginMeta` component. Have a Jetpack site selected and go to `plugins/jetpack`. The page has the toggles in the header area:

![plugin-meta](https://user-images.githubusercontent.com/664258/29861225-d99db102-8d68-11e7-9368-6301f68dc371.gif)

Second place is the `PluginSite` component. Select "All My Sites" in the site selector and to to `/plugins/amp`. At the bottom, there is a section for each site that has the plugin installed:

![plugin-site](https://user-images.githubusercontent.com/664258/29861304-2268f5b8-8d69-11e7-8308-81e8484d4bb3.gif)

That's why I was adding the style overrides for `.plugin-action` et al. as a nested selector in `plugin-item/style.scss`:

```css
.plugin-item {
  .plugin-action {
    ...
  }
}
```

Reply:
Oh, I see! I'll fix it as well as the plugin icon you mentioned below. Sorry about that.